workflows/diagnostics/fv3net/diagnostics/offline/compute.py

```diff
@@ -310,7 +309,7 @@ def main(args):
         grid = load_grid_info(args.grid_resolution)
     else:
         with fsspec.open(args.grid, "rb") as f:
-            grid = xr.open_dataset(f).load()
+            grid = xr.open_dataset(f, engine="h5netcdf").load()
```

Comment:
Now need to use this engine for opening a local fsspec filesystem object (maybe an xarray regression?)